scripts/build/builders/ameba.py

```diff
@@ -51,6 +51,13 @@ def AppNamePrefix(self):
             raise Exception('Unknown app type: %r' % self)
 
 
+def OptionalFlags(self):
```

Comment:
`self` is generally expected to be a member method. Please move as member method (call as self.OptionalFlags) and also fix naming - what is an optional flag? What is its use? what makes pigweed-app special?

From this context, I do not know what `-r` acutally means.

Reply:
The optionalflag function is meant to send some flags to platform side for compilation.
The -r stands for rpc.